# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-contain/content-visibility/content-visibility-070.html.ini
# Commit: c1e64e8e4665
# Full Hash: c1e64e8e466506d1d9a36d747df25527bd545b10
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-12-01 16:18:29
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - Add support for `content-visibility: auto` r=emilio
#   
#   This change adds support for `content-visibilty: auto` as well as
#   showing and hiding content based on the relevancy of the content as
#   defined in the specification. Changes to relevancy are handled by
# ==============================================================================

diff -r 6d4d56f57b7b -r c1e64e8e4665 testing/web-platform/meta/css/css-contain/content-visibility/content-visibility-070.html.ini
--- a/testing/web-platform/meta/css/css-contain/content-visibility/content-visibility-070.html.ini	Thu Dec 01 01:23:08 2022 +0200
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,5 +0,0 @@
-[content-visibility-070.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [Content Visibility: off-screen selection]
-    expected: FAIL